src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const allPopulationCollection = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+const peopleSum = [ ...allPopulationCollection ]
+  .reduce((prev, item) => {
+    let number = '';
+
+    for (const numberPart of item.textContent) {
+      if (numberPart !== ',') {
+        number += numberPart;
+      }
+    }
```

Comment:
Could be simplify 
![image](https://github.com/mate-academy/js_get_data_DOM/assets/70009205/1d46bc8e-e7aa-43ad-87ae-7d24f02420bc)